typescript/cli/anvil-start.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
No problem with this little script but I'm wondering who it's intended for? If you intend end-users to execute it, it would need to be exposed via the package.json's bin field. It's finicky but doable.

Reply:
Through the course of development this script became no longer utilized– but yes it was initially intended for end-users to run anvil. I'll remove it to reduce clutter as it doesn't seem worth exposing via bin